### ISSUE TITLE
Cannot get it to work

### ISSUE THREAD

**Issue by AussieGit** [darktrojan/openwith#183]
Loved the Open With prior to FF57 destroying everything. Have tried all the extra steps, downloading version 2 and 3 of Python, ensured it is in my patch, saved the install file as per step 2, then gone into a command prompt all to no avail!! Testing Installation, I keep getting the message "Something went wrong. There might be more information in the Browser Console." but I have no clue about that.

As an aside, if it comes to Browser concoles and Command prompts to get an extension working which I am sure most don't know about then Firefox is wrecking this and other great extensions.

**Comment by darktrojan** [comment]
I agree that having to go to these lengths isn't ideal but at least it is still possible to have things like Open With. Google designed this system, by the way, so Firefox isn't solely to blame.

There's some good info about the Console messages in #173, and I've just made an illustrated guide in #184 that might be helpful too.

**Comment by mbooga** [comment]
For windows, this can be helpful:
https://github.com/mbooga/owclauncher
